client-docs/README.md

```diff
@@ -161,26 +167,38 @@ Emits when the list of local peers updates (new peers are discovered, or a peer'
 
 Namespace for managing project invites. Provides an event emitter-like interface so it can emit and subscribe to events.
 
+##### `invite.getPending`
+
+`() => Promise<PendingInvite[]>`
+
+Returns a list of pending invites.
+
 ##### `invite.accept()`
 
-`(projectId: string) => Promise<MapeoProject>`
+`(inviteId: PendingInvite['inviteId']) => Promise<MapeoProject>`
 
-Accept an invitation that was received for the project associated with `projectId`.
+Accept an invitation that was received using the `inviteId`. Returns the `MapeoProject` that has just been joined by accepting the invite.
 
 ##### `invite.reject()`
 
-`(projectId: string) => Promise<void>`
+`(inviteId: PendingInvite['inviteId']) => Promise<void>`
 
-Reject an invitation that was received for the project associated with `projectId`.
+Rejects an invitation that was received using the `inviteId.
 
 ##### Events
 
 ###### `'invite-received'`
 
-`invite.addEventListener('invite-received', (invite: Invite) => void)`
+`invite.addEventListener('invite-received', (invite: PendingInvite) => void)`
 
 Emits when a device invites you to a project.
 
+###### `'invite-removed`
+
+`invite.addEventListener('invite-received', (inviteWithReason: PendingInvite & {reason:'cancelledByInvitor' | 'rejectedByInvitee'}) => void)`
```

Comment:
Im open to suggestions for the wording of the `reason` type. Its a little verbose right now, but not necessarily a problem

Reply:
1. I generally prioritize clarity over brevity, so I like these `reason` names.
1. Invites can be removed because they're rejected by the invitee, but also because they're _accepted_ by the invitee. They can also be removed because of a timeout.
1. You wrote "invite-received" here when I think you meant to write "invite-removed".

```suggestion
`invite.addEventListener('invite-removed', (inviteWithReason: PendingInvite & {reason:'cancelledByInvitor' | 'removedByInvitee' | 'timeout'}) => void)`
```

Could also be worth updating the note below.